pipelines/tekton/azureml-container-pipeline/model-upload/basic-pod.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: basic-pod
+spec:
+  volumes:
+    - name: PVC_NAME
```

Comment:
_"I know this is just a placeholder, but it's not a valid name for an object in Kubernetes, because of the `_`s. See here for more info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-subdomain-names"_ - @grdryn circa 2023 (colorized) ([taken from here](https://github.com/opendatahub-io/ai-edge/pull/55#discussion_r1315092519)) (not sure if this one still applies here)

Reply:
Done 👍  changed to just `PVC`